stdlib/public/Concurrency/TaskGroup.cpp

```diff
@@ -2097,10 +2097,12 @@ static bool swift_taskGroup_isCancelledImpl(TaskGroup *group) {
 
 SWIFT_CC(swift)
 static void swift_taskGroup_cancelAllImpl(TaskGroup *group) {
-  asBaseImpl(group)->cancelAll();
+  // TaskGroup is not a Sendable type, so this can only be called from the
+  // owning task.
+  asBaseImpl(group)->cancelAll(swift_task_getCurrent());
```

Comment:
Hm yeah that's true today. It's fine to make this explicit here this way.

Reply:
This comment is repeated in a few places where we make this assumption, so I figured I'd put it here too.